dev-docs/best-practices/ui.md

```diff
@@ -27,6 +28,14 @@ When we have a good reason to not follow the best practices in the UI, we docume
 This should be the close modal button. This will help prevent users
 from accidentally triggering a destructive action.
 
+## Action Buttons
+
+A button that performs an action (i.e., sends a request to the backend) will most likely require a
```

Comment:
Maybe this should say "i.e. mutates state on the backend" or something since gets are mostly fine? I'm not 100% contextually knowledgeable here though so feel free to ignore this.

Reply:
Agreed. Thanks!